pkg/manifest/manifest.go

```diff
@@ -29,6 +30,10 @@ type StatusStanza struct {
 	LocalhostOnly bool   `yaml:"localhost_only,omitempty"`
 }
 
+type ResourceLimitsStanza struct {
+	CGroup cgroups.Config `yaml:"cgroup,omitempty"`
```

Comment:
Unfortunately using the `cgroups.Config` type directly, because it has a `size.ByteCount` with a `MarshalYAML()` function that marshals it to an integer number of bytes.

In other words using this type will mean that pod manifests will always have an integer number of bytes instead of something human readable like `10G`. I'm not really sure what to do about this though, we want it to marshal differently when it's in a pod manifest vs when it's in a cgroup file

Reply:
You're more of a cgroups expert than I am, but i'm used to hearing it pluralized like "cgroups". Would that be more appropriate for the YAML key?